### DISCUSSION TITLE
People with performance issues and Forever running queues with no output look here [FLUX]

### DISCUSSION THREAD

**Discussion by BrechtCorbeel** [Comfy-Org/ComfyUI#4730] [General]
So I've done a lot of experimenting, this is not a guide or full description of what is or are issues, but I have come to some conclusions:

(note I am talking 4090, I have a 3060 too and a 1070, what might run but very slow on a lower end VRAM card can bug out a high VRAM card)

- fp8 clip can handle less characters then fp16 (CLIP).
- fp16 will still result in issues with long prompts even though it will run much more workflows before doing so.
- I had prompts of 20.000 characters that would perpetually get stuck and give VRAM errors + make my GPU run infinitely,
I ran the same prompts after reducing the characters to 10.000 at some point I still got the VRAM error that normally would bug out my GPU but at 8.000 it still ended up running flirting with the maximum VRAM, but still running and at a normal speed so do not go over 8.000 - 9.000 characters. 10.000 works fine for a while but still causes issues.
UPDATE: after some more testing it was only when I went as low as 6000 character that the issue completely disappeared (for now)
- batches can increase VRAM, seems counter intuitive, but some nodes do batches stored in VRAM.
- I mainly run 1536 x 1024 on my 4090 upscaling afterwards, schnell can do much higher res, but the images are overly saturated and lack substance.
- additionally I think it does not like it if you have \n written in your prompts or some other coding related character sets.

**Comment by BrechtCorbeel** [comment]
additionally I think it does not like it if you have \n written in your prompts or some other coding related character sets.

**Comment by JorgeR81** [comment]
You can try the T5 text encoder, in Q8_0 format, via this custom node.
https://github.com/city96/ComfyUI-GGUF

It's a smaller file, so it takes less VRAM.
It's not as good as FP16, but it should be better than FP8.
https://huggingface.co/city96/t5-v1_1-xxl-encoder-gguf/tree/main

You can also try Flux in Q8_0, to save additional VRAM.
https://huggingface.co/city96/FLUX.1-dev-gguf/tree/main
https://huggingface.co/city96/FLUX.1-schnell-gguf/tree/main
